applications/prompt-proto-service-lsstcomcam/values-usdfprod-prompt-processing.yaml

```diff
@@ -0,0 +1,74 @@
+prompt-proto-service:
+
+  podAnnotations:
+    # HACK: disable autoscaling as workaround for DM-41829
+    autoscaling.knative.dev/min-scale: "20"
+    autoscaling.knative.dev/max-scale: "20"
+    # Update this field if using latest or static image tag in dev
+    revision: "1"
+
+  # HACK: disable autoscaling as workaround for DM-41829
+  worker:
+    restart: 15
+
+  image:
+    pullPolicy: IfNotPresent
+    # Overrides the image tag whose default is the chart appVersion.
+    tag: 4.5.1
+
+  instrument:
+    pipelines:
+      # BLOCK-306 is photographic imaging
+      # BLOCK-T17 is daytime checkout
+      # BLOCK-271 is photon transfer curve calibrations
+      # BLOCK-295 is the daily calibration sequence as of May 27, 2024
+      main: >-
+        (survey="BLOCK-306")=[${PROMPT_PROCESSING_DIR}/pipelines/LSSTComCam/ApPipe.yaml,
```

Comment:
These survey names are for LATISS and are unlikely to be the same for LSSTComCam. See the [Slack thread ](https://rubin-obs.slack.com/archives/C07QJMQC8KC/p1729703256595719). It's okay to put in a simpler placeholder for now.